cmd/kubeadm/app/phases/certs/renewal/manager_test.go

```diff
@@ -305,3 +324,188 @@ func makeTestCertConfig(organization []string) *pkiutil.CertConfig {
 		},
 	}
 }
+
+func TestManagerCAs(t *testing.T) {
+	tests := []struct {
+		name string
+		cas  map[string]*CAExpirationHandler
+		want []*CAExpirationHandler
+	}{
+		{
+			name: "CAExpirationHandler is sequential",
+			cas: map[string]*CAExpirationHandler{
+				"foo": {
+					Name: "1",
+				},
+				"bar": {
+					Name: "2",
+				},
+			},
+			want: []*CAExpirationHandler{
+				{
+					Name: "1",
+				},
+				{
+					Name: "2",
+				},
+			},
+		},
+		{
+			name: "CAExpirationHandler is in reverse order",
+			cas: map[string]*CAExpirationHandler{
+				"foo": {
+					Name: "2",
+				},
+				"bar": {
+					Name: "1",
+				},
+			},
+			want: []*CAExpirationHandler{
+				{
+					Name: "1",
+				},
+				{
+					Name: "2",
+				},
+			},
+		},
+	}
+	for _, tt := range tests {
+		t.Run(tt.name, func(t *testing.T) {
+			rm := &Manager{
+				cas: tt.cas,
+			}
+			if got := rm.CAs(); !reflect.DeepEqual(got, tt.want) {
+				t.Errorf("Manager.CAs() = %v, want %v", got, tt.want)
+			}
+		})
+	}
+}
+
+func TestManagerCAExists(t *testing.T) {
+	certificateReadWriterexist := fakecertificateReadWriter{
```

Comment:
```suggestion
	certificateReadWriterExist := fakecertificateReadWriter{
```

please use camel case properly.